app/models/user.rb

```diff
@@ -63,4 +63,8 @@ def telegram?
   def email?
     email.present?
   end
+
+  def replies_by_request
+    replies.eager_load(:request).reorder('requests.created_at DESC, messages.created_at ASC').group_by(&:request)
```

Comment:
I think the requests feed should be ordered by the date/time a user has replied/interacted, right? Also, intuitively, I though it would make sense to display the date the users replied the last time (not the first time).

Example:
1. An editor creates a new request on Monday, 8am.
2. A user replies on Monday, 9am.
3. The same user replies a second time on Tuesday, 10am.

Then in that user’s activity feed, I should see:

> has replied to {Request} on Tuesday, 10am.

instead of

> has replies to {Request} on Monday, 9am.